target/linux/realtek/image/rtl930x.mk

```diff
@@ -114,6 +114,24 @@ define Device/zyxel_xgs1210-12-a1
 endef
 TARGET_DEVICES += zyxel_xgs1210-12-a1
 
+define Device/zyxel_xgs1210-12-b1
+  SOC := rtl9302
+  SUPPORTED_DEVICES += zyxel,xgs1210-12
+  UIMAGE_MAGIC := 0x93001210
+  ZYXEL_VERS := ABTY
+  DEVICE_VENDOR := Zyxel
+  DEVICE_MODEL := XGS1210-12
+  DEVICE_VARIANT := B1
+  IMAGE_SIZE := 13312k
+  KERNEL_INITRAMFS := \
+        kernel-bin | \
+        append-dtb | \
+        gzip | \
+        zyxel-vers | \
+        uImage gzip
+endef
+TARGET_DEVICES += zyxel_xgs1210-12-b1
```

Comment:
Have a look at PR for XGS1250 B2 how to deduplicate both recipes.

Reply:
P.S. give "lzma" and "uImage lzma" a try to save 1MB of space.

Reply:
done